src/providers/ipa/ipa_views.c

```diff
@@ -456,6 +566,16 @@ static void ipa_get_ad_override_done(struct tevent_req *subreq)
         state->dp_error = DP_ERR_OK;
         tevent_req_done(req);
         return;
+    } else if (reply_count == 2 &&
```

Comment:
Hi,

I would like to suggest to use `REPLIES`, I would also suggest to rename it to e.g. `MAX_USER_AND_GROUP_REPLIES` to make more clear what this number is about.

bye,
Sumit

Reply:
Done.